troposphere/static/js/components/mixins/ChosenMixinExternal.jsx

```diff
@@ -81,6 +81,12 @@ export default {
     onEnter: function(e) {
         if (e.which !== ENTER_KEY) return;
 
+        // Stops the enter event from bubbling out to other elements.
+        // If we want to say move focus to the another field on enter, a line break (\n)
+        // would be passed to that field causing the line break to be added before the cursor.
+        e.preventDefault();
+        e.stopPropagation();
+
```

Comment:
Glad to see a reference; it'll be helpful in preventing someone from removing it.

- [MDN preventDefault](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)
- [MDN stopPropagation](https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation)